static/app/utils/discover/fields.tsx

```diff
@@ -843,6 +845,15 @@ export function isAggregateEquation(field: string): boolean {
   return isEquation(field) && results !== null && results.length > 0;
 }
 
+export function isEquationColumn(column: Column): boolean {
```

Comment:
The name of this function seems misleading? It's really testing if a column is allowed in an equation rather than the column being an equation.

Reply:
went with isLegal to match the yAxis Function